sdk/template/ci.yml

```diff
@@ -33,10 +33,21 @@ pr:
     - eng/common/
 #endif
 
+# Switch to canary to test canary 1es branch. 1es template validation will set this parameter
+# to canary on run.
+parameters:
+- name: oneESTemplateTag
+  type: string
+  default: release
+  values:
+    - release
```

Comment:
Shouldn't we swap this? So it's `canary` by default?

Reply:
(because this is specific to template)

Reply:
I think for normal operations we want template on release (since we'll be using template to test other things). When 1es kicks it off on their side for validation, I expect the parameter to be overridden to canary.

Reply:
Ahh they'll actually provide that parameter? Got it.